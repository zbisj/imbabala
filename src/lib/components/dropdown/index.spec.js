import React from "react";
import renderer from "react-test-renderer";
import "jest-styled-components";

import Dropdown from ".";

describe("Dropdown", () => {
  test("renders correctly", () => {
    const tree = renderer
      .create(
        <Dropdown
          label="Click Me"
          items={[
            {
              label: "Item 1",
            },
          ]}
          variant="warning"
        />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
