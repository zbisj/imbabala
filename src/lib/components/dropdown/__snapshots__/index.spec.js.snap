// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dropdown renders correctly 1`] = `
<button
  className=""
  onClick={[Function]}
>
  Click Me
</button>
`;
